src/content/1.7/development/webservice/tutorials/1-creating-access.md

```diff
@@ -66,7 +68,7 @@ $apiAccess->save();
 This first code allows you to pass the authentication layer. You also need access to the resources you expect to use.
 
 We need the Api account ID in order to grant it access, and an array having the resource name as key and the array of methods allowed as value.
-The available resources can be found in [`WebserviceRequest::getResources()` (link to definition)](https://github.com/PrestaShop/PrestaShop/blob/844ceb99b26f0b81c4f96b58a304fd3dbe27de41/classes/webservice/WebserviceRequest.php#L285]).
+The available resources can be found in [`WebserviceRequest::getResources()` (link to definition)](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/webservice/WebserviceRequest.php#L285]).
```

Comment:
Don't use links to develop because they change, choose a version tag instead:

```suggestion
The available resources can be found in [`WebserviceRequest::getResources()` (link to definition)](https://github.com/PrestaShop/PrestaShop/blob/1.7.6.0/classes/webservice/WebserviceRequest.php#L285]).
```